tests/transports-integrations/tests/integrations/test_anthropic.py

```
@@ -0,0 +1,589 @@
+                            {
+                                "type": "image",
+                                "source": {
+                                    "type": "base64",
+                                    "media_type": "image/png",
+                                    "data": data,
+                                },
+                            }
+                        )
+                    else:
+                        # URL image - need to download and convert to base64
+                        response = requests.get(url)
```

Ruff (0.11.9)

Probable use of `requests` call without timeout

(S113)